docs/custom-resource.md

```diff
@@ -164,8 +164,8 @@ This tables shows the component types and the client secrets they use.
 | `content-server`/`rls` | `replication` |             |           | `publisher` | To the MLS  |
 | `cae`/`live`           |               | `blueprint` | `live`    | `webserver` |             |
 | `cae`/`preview`        |               | `blueprint` | `preview` | `webserver` |             |
-| `cae-feeder`/`live`    | `caefeeder`   | `blueprint` | `live`    | `webserver` |             |
-| `cae-feeder`/`preview` | `mcaefeeder`  | `blueprint` | `preview` | `webserver` |             |
+| `cae-feeder`/`preview` | `caefeeder`   | `blueprint` | `preview` | `webserver` |             |
```

Comment:
Thanks for the correction. Would you mind swapping the two lines, so the entries in the table are all in alphabetical order?